opcua/server/server.py

```diff
@@ -641,3 +644,42 @@ def set_attribute_value(self, nodeid, datavalue, attr=ua.AttributeIds.Value):
         so it is a little faster
         """
         return self.iserver.set_attribute_value(nodeid, datavalue, attr)
+    
+    def set_user_manager(self, user_manager):
```

Comment:
I tend to think these methods should be part of InternalServer. They really implement new feature. The Server class is supposed to only be a high level helper class

Reply:
The reason the user_token_manager function (or check_user_token) used in InternalServer is in Server is that it needs access to the Server.private_key to decode the token password. Any suggestion on how to gain access to this variable from InternalServer?

Reply:
It is probably wrong to store private_key in server. Can you move it to internal server?